### PR TITLE
SM: connected log in with other pages

### PR DESCRIPTION
**Pull request by smathis122**

### CODE REVIEW

File: code/Frontend/react-frontend/src/App.js [hunk 1]
```diff
@@ -1,47 +1,74 @@
 import React from "react";
-import { HashRouter, BrowserRouter, Routes, Route } from "react-router-dom";
+import { HashRouter, Route, Routes } from "react-router-dom";
```
Comment (casejade37) [inline code/Frontend/react-frontend/src/App.js:2]: Good practice - we need to follow this pattern to remove everything not used

File: code/Backend/flask-backend/app.py [hunk 2]
```diff
@@ -256,7 +256,7 @@ def login():
         conn = psycopg2.connect(**db_connection_settings)
         cursor = conn.cursor()
         form = LoginForm(meta={'csrf': False})
-
+        session['username'] = None
```
Comment (casejade37) [inline code/Backend/flask-backend/app.py:259]: Is 'session' already initialized by Flask?

File: code/Frontend/react-frontend/src/pages/AddItem.js [hunk 3]
```diff
@@ -1,16 +1,18 @@
 import React, { useState } from "react";
-import { NavbarCustom } from "../Components/navbar";
+import { NavbarCustom } from "../Components/Navbar";
```
Comment (casejade37) [inline code/Frontend/react-frontend/src/pages/AddItem.js:2]: Nice renaming to follow convention

File: code/Frontend/react-frontend/src/pages/search.js
Comment (casejade37) [inline code/Frontend/react-frontend/src/pages/search.js:1]: What is changed in this file? I assume the diff showed here are just formatting?

**Review by casejade37 — APPROVED** [review]
Good job! I have a few questions in the review but I don't think any of them will cause runtime errors - so this is good to go.

File: code/Frontend/react-frontend/src/pages/search.js
Comment (smathis122) [reply code/Frontend/react-frontend/src/pages/search.js:1]: Yeah I didn't touch this file must just be git styling stuff

File: code/Backend/flask-backend/app.py (hunk 2 above)
Comment (smathis122) [reply code/Backend/flask-backend/app.py:259]: No i don't think so - I had to add it to the import at the top

**Review by ahnaft7 — APPROVED** [review]

**Review by ahnaft7 — COMMENTED** [review]
Noticed clicking the GearOnTheGo Logo logs the user out. Was this intentional or just something to change later?

**Comment by smathis122** [comment]
> Noticed clicking the GearOnTheGo Logo logs the user out. Was this intentional or just something to change later?

Not intentional I think it's cause it redirects you to the home page it reloads the page and logs you out -- I didn't make any edits or modifications to log out. I figured that was a different story

**Pull request merged**